### PR TITLE
feat: add njxqlus progressbar plugin

### PR DESCRIPTION
**Pull request by njxqlus**

### CODE REVIEW

File: content/plugins/images/njxqlus-progressbar.jpg
Comment (danharrin) [inline content/plugins/images/njxqlus-progressbar.jpg:1]: Please add some text or something to the image so it doesnt look so empty

**Review by danharrin — CHANGES_REQUESTED** [review]

File: content/authors/njxqlus.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Mikael Ahabalyants 
+slug: njxqlus
+avatar: njxqlus.jpg
+github_url: https://github.com/njxqlus
+---
```
Comment (danharrin) [inline content/authors/njxqlus.md:6]: Please add a bio below this author information, as in our example

File: content/plugins/images/njxqlus-progress-bar.jpg
Comment (danharrin) [inline content/plugins/images/njxqlus-progress-bar.jpg:1]: I appreciate the effort that was put in here, however the design does look quite different to the other plugins on our website, and that contrast might put users off from clicking on it. Maybe you could have a look at the other plugins that are on our website for style inspiration? It's up to you if you want to redesign it or not, please let me know if you want to proceed and I will publish it as-is.

Comment (njxqlus) [reply content/plugins/images/njxqlus-progress-bar.jpg:1]: Sorry, but I don't see the common style of these plugins. Maybe I could change it later?

File: content/authors/njxqlus.md (hunk 1 above)
Comment (njxqlus) [reply content/authors/njxqlus.md:6]: Updated

**Comment by njxqlus** [comment]
> Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.

Can you check if I have done it right, please?

**Pull request merged**